api-spec/client/openapi/redirect/redirect-api.yaml

```diff
@@ -152,27 +150,32 @@ components:
           minLength: 1
           maxLength: 70
         idPaymentMethod:
-          description: Unique id of the payment method chosen by the user
+          description: Redirect payment method type
           type: string
-          format: uuid
+          enum:
```

Comment:
are you sure we want to have this enum? potentially every PSP could have its own domain

Reply:
Fixed with latest push 👌